models/GATStack.py

```diff
@@ -33,7 +33,7 @@ def __init__(
         self.convs = ModuleList()
         self.batch_norms = ModuleList()
         self.convs.append(
-            GATConv(
+            GATv2Conv(
```

Comment:
@pzhanggit @streeve 
I just read the description of the GATv2Conv layer in `torch.geometric`